### ISSUE TITLE
CVPR 2018 Reading List 

### ISSUE THREAD

**Issue by jodoin** [vitalab/vitalab.github.io#172]
### MUST READ
# Sarah

- [x] Learning From Synthetic Data: Addressing Domain Shift for Semantic Segmentation
- [x] Regularizing Deep Networks by Modeling and Predicting Label Structure
- [x] Anatomical Priors in convolutional neural networks for unsupervised Biomedical segmentation
- [x] Learning time memory- efficient deep architecture with Budgetted Super network

# Carl

- [x] Practical Block-Wise Neural Network Architecture Generation (note by Carl: similar to ENAS but very worse)
- [x] Learning-Compression Algorithms for Neural Net Pruning
- [x] BlockDrop: Dynamic Inference Paths in Residual Networks
- [x] PackNet : Adding Multiple Tasks to a single Network by Iterative Pruning
- [x] Clip-Q : Deep network compression leanring gby in-parallel Pruning Quantization
- [x] NISP : Pruning Networks using Neuron importance Score Propagation

# Faezeh

- [x] Beyond the Pixel-Wise Loss for Topology-Aware Delineation
- [x] Interactive image segmentation with latent diversity
- [x] Quantization fo Fully Convolutional Networks for accurate Biomedical image segmentation

# Daniel

- [x] Burst Denoising With Kernel Prediction Networks
- [x] Squeeze-and-Excitation Networks (2017 imageNet winner)

# Charles

- [x] Deep extreme cut
- [ ] Functional Map of the World
- [ ] ClusterNet: Detecting Small Objects in Large Scenes by Exploiting Spatio-Temporal Information
- [x] DOTA: A Large-Scale Dataset for Object Detection in Aerial Images
- [x] Deeply Learned Filter Response Functions for Hyperspectral Reconstruction
- [x] Context Encoding for Semantic Segmentation

# Fred

- [x] What do deep networks like to see
- [x] Rethinking the Faster R-CNN Architecture for temporal action localization
- [x] Wasserstein Introspective Neural Networks
- [x] Cascade R-CNN: Delving Into High Quality Object Detection

# PM
- [x] Normalized cut loss for weakly-supervised cnn segmentation
- [ ] Weakly Supervised Instance Segmentation Using Class Peak Response
- [x] Classification-Driven Dynamic Image Enhancement

# Everyone

- [ ] Context Contrasted Feature and Gated Multi-Scale Aggregation for Scene Segmentation
- [x] Deep Layer Aggregation
- [x] Convolutional Neural Networks With Alternately Updated Clique
- [x] Light weight Probabilistic Deep Network
- [ ] Revisiting Knowledge transfer for training Object class detectors
- [ ] Efficient Optimization for Rank-Based Loss Functions
- [x] Taskonomy: Disentangling Task Transfer Learning (BEST PAPER AWARD)
- [x] Learning to segment everything
- [ ] Few-Shot Image Recognition by Predicting Parameters From Activations
- [x] Revisiting Dilated Convolution: A Simple Approach for Weakly- and Semi-Supervised Semantic Segmentation

### SUGGESTED

- [x] xUnit: Learning a Spatial Activation Function for Efficient Image Restoration
- [ ] Residual Dense Network for Image Super-Resolution
- [x] Attentive Generative Adversarial Network for Raindrop Removal From a Single Image
- [x] Efficient Interactive Annotation of Segmentation Datasets With Polygon-RNN++
- [x] Learning Steerable Filters for Rotation Equivariant CNNs
- [ ] Disentangling structure and aesthetics for style-awareimage completion
- [x] Learning discriinative feature network for semantic segmentation
- [ ] Defense againts adversarial attack using high-level representation guided denoiser
- [x] SeedNet: Automatic seed generation with Deep reinforcement learning for Robust Interactive segmentation
- [ ] CLEAR: Cumulative LEARning for One Shot One Class Image Recognition
- [ ] Collaborative and Adversarial Network for Unsupervised Domain Adaptation
- [x] MegDet: A Large Mini-Batch Object Detector
- [ ] Semantic Video Segmentation by Gated Recurrent Flow Propagation
- [ ] Dense Decoder Shortcut Connections for Single-Pass Semantic Segmentation
- [ ] CosFace: Large Margin Cosine Loss for Deep Face Recognition
- [x] MaskLab: Instance Segmentation by Refining Object Detection With Semantic and Direction Features
- [ ] Learning Pixel-Level Semantic Affinity With Image-Level Supervision for Weakly Supervised Semantic Segmentation
- [ ] Few-Shot Image Recognition by Predicting Parameters From Activations
- [ ] CVM-Net: Cross-View Matching Network for Image-Based Ground-to-Aerial Geo-Localization
- [ ] Low-Shot Learning From Imaginary Data
- [x] Guide Me: Interacting With Deep Networks
- [ ] Interleaved Structured Sparse Convolutional Neural Networks
- [ ] Interpretable convolutional neural networks
- [ ] Detail-Preserving Pooling in Deep nets
- [ ] Efficient parametrization of multi-domain deep neural nets
- [ ] Deep image prior
- [ ] Learning nested sparse Structures in Deep neural nets
- [ ] The Power of Ensemble for Active LEarning in image classification

### PRE-2018 PAPERS
- [x] Dilated Residual Networks
- [x] Simple Does It: Weakly Supervised Instance and Semantic Segmentation
- [ ] Deep Interactive Object Selection
 
### SPECTRAL CLUSTERING STUFF
- [ ] Fast spectral ranking for similarity search
- [x] Mining on Manifolds: metric learning without labels
